drivers/counter/Kconfig.mcux_gpt

```diff
@@ -7,4 +7,6 @@ config COUNTER_MCUX_GPT
 	bool "MCUX GPT driver"
 	depends on HAS_MCUX_GPT
 	help
-	  Enable support for mcux General Purpose Timer (GPT) driver.
+	  Enable support for mcux General Purpose Timer (GPT) driver. Note that
+	  enabling the mcux GPT as a counter will disable using it for the hardware
+	  clock, and the platform will fall back to using the systick timer
```

Comment:
`, and the platform will fall back to using the systick timer` I think this should be deleted as the current design may not have this.